README.md

```diff
@@ -1,5 +1,5 @@
-<!-- ![](https://img.shields.io/badge/Microverse-blueviolet) -->
-![This is an image](./images/check.jpg)
+
```

Comment:
- [ ] Kindly fix the image because it is not working. Keep shining 💪 

Here you have a screenshot from your Readme file:

![image](https://user-images.githubusercontent.com/74506933/159078594-6879bc26-df08-40b1-96d1-d5c37083c051.png)